types/webpack/index.d.ts

```diff
@@ -1073,9 +1073,13 @@ declare namespace webpack {
           hooks: {
             jsonpScript?: SyncWaterfallHook<string, Chunk, string>;
             requireExtensions: SyncWaterfallHook<string, Chunk, string>;
+            requireEnsure: SyncWaterfallHook<string, Chunk, string>;
+            localVars: SyncWaterfallHook<string, Chunk, string>;
           };
           outputOptions: Output;
           requireFn: string;
+          renderRequireFunctionForModule(hash: string, chunk: Chunk, varModuleId: number | string): string;
+          renderAddModule(hash: string, chunk: Chunk, varModuleId: number | string, varModule: string): string;
```

Comment:
The param `varModuleId` is marked as optional in the JSDoc annotation:
https://github.com/webpack/webpack/blob/v4.41.5/lib/MainTemplate.js#L480

Reply:
Had to change this one to:

```ts
renderAddModule(hash: string, chunk: Chunk, varModuleId: number | string | undefined, varModule: string): string;
```

but from the code it looks like passing undefined there would render something like `module[undefined] = myModule;`. Anyway, I assume following the JSDoc is more preferable?

Reply:
Yeah I agree with that it makes not much sense for this param here.
But since it doesn't break anything here because it is only used to generate a string, I think it's safe to follow the JSDoc here.